draft-ietf-quic-recovery.md

```diff
@@ -1122,10 +1122,10 @@ A receiver can misreport ECN markings to alter the congestion response of a
 sender.  Suppressing reports of ECN-CE markings could cause a sender to
 increase their send rate.  This increase could result in congestion and loss.
 
-A sender MAY attempt to detect suppression of reports by marking occasional
-packets that they send with ECN-CE.  If a packet sent with ECN-CE is not
+A sender can detect suppression of reports by marking occasional packets that it
```

Comment:
Most, if not all of this whole section seems like it should be in transport, since it discusses disabling ECN?

Reply:
The attack is on the congestion controller, so it's fine to have it here I think.

Reply:
I agree with Jana.  The transport doc has a long enough security considerations section.

The same is not true for the previous section (on Traffic Analysis).